doc/install.rst

```diff
@@ -452,10 +452,12 @@ Sphinx
 We use Sphinx to build the documentation; the theme is
 `sphinx-rtd-theme <https://sphinx-rtd-theme.readthedocs.io/en/stable/>`_.
 
-Minimum versions are listed above. We currently use Sphinx 1.8 for building
+Minimum versions are listed above. We currently use Sphinx 3.2.1 for building
 what's on the web because there are bugs problematic for us in 2.x, e.g. `bad
 spacing in lists
-<https://github.com/readthedocs/sphinx_rtd_theme/issues/799>`_.
+<https://github.com/readthedocs/sphinx_rtd_theme/issues/799>`_ and versions
+prior to 2.x will `return markdown when searching
+<https://github.com/sphinx-doc/sphinx/issues/1618>`_.
```

Comment:
I wonder if something like:

> Minimum versions are listed above. Note that while anything greater than the minimum should yield readable documentation, we don't test quality with anything other than what we use to build the website, which is usually but not always the most recent version available on PyPI.

would be clearer and more future-proof. E.g., saying we use 3.x because 2.x is buggy seems a bit odd.

Reply:
I really like this wording, I will add it.